execution/evm/state.go

```diff
@@ -182,7 +203,7 @@ func (st *State) CreateAccount(address crypto.Address) {
 	st.updateAccount(&acm.Account{Address: address})
 }
 
-func (st *State) InitCode(address crypto.Address, code []byte) {
+func (st *State) InitCode(address crypto.Address, parent *crypto.Address, code []byte) {
```

Comment:
All usages of `InitCode` except one pass nil for the second argument. I tend to think this is a smell that indicates that we should introduces two methods and share implementation.

Can we introudce `InitChildCode` or similar that takes the parent argument, does the forebear bit, then calls `InitCode`?

We can also drop the unnecessary pointer then

Reply:
Good point; fixed